precompose/src/commonMain/kotlin/moe/tlaster/precompose/navigation/NavHost.kt

```diff
@@ -240,6 +240,7 @@ fun NavHost(
                 DragSlider(
                     state = state,
                     enabled = prevSceneEntry != null,
+                    spaceToSwipe = actualSwipeProperties?.spaceToSwipe ?: 10.dp,
```

Comment:
Actually, `actualSwipeProperties` can't be null when we are checking `state != null`

So maybe worth to make it `if (state != null && actualSwipeProperties != null)`

Reply:
Yes that make more sense